Project_Mars/Project_Mars/StepDefinitions/SkillStepDefinitions.cs

```diff
@@ -0,0 +1,80 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+
+namespace Project_Mars.StepDefinitions
+{
+    [Binding]
+    [Parallelizable]
+    public class SkillStepDefinitions
+    {
+
+        LoginPage LoginPageobj = new LoginPage();
+        HomePage HomePageobj = new HomePage();
+        SkillTab SkillTabobj = new SkillTab();
+        private IWebDriver driver;
+
+        public SkillStepDefinitions(IWebDriver driver)
+        {
+           this.driver = driver;
```

Comment:
same comment about constructors here

Reply:
who is passing the driver now as the framework is creating this object?

Reply:
Removed this